<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>histogramDialog</class>
 <widget class="QDialog" name="histogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogram</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
   <widget class="QComboBox" name="rCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result &amp;Collector:</string>
    </property>
    <property name="buddy">
     <cstring>rCombo</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result &amp;Data Source:</string>
    </property>
    <property name="buddy">
     <cstring>dCombo</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="dCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAuto">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Automatic Rebuild</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Histogram</string>
   </property>
   <widget class="QLabel" name="progressBarMsg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Close Tool Window</string>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRebuild">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Update Plot</string>
   </property>
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Save Plot to Image File...</string>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Bins:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bandSpin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>16</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Custom</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Auto</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="aCombo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram Colors:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QToolButton" name="colorChangeButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCAuto">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Auto</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCCustom">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Single Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblColorValue">
     <property name="geometry">
      <rect>
       <x>358</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCGray">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>0</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Grayscale</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Save Plot to Image File...</string>
   </property>
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rCombo</tabstop>
  <tabstop>dCombo</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>aCombo</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>bandSpin</tabstop>
  <tabstop>btnRebuild</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>aCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>bandSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorChangeButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
